modules/nf-core/mageck/test/main.nf

```diff
@@ -0,0 +1,38 @@
+process MAGECK_TEST {
+    tag "$meta.id"
+    label 'process_medium'
+
+    conda "bioconda::mageck=0.5.9"
+    container "${ workflow.containerEngine == 'singularity' && !task.ext.singularity_pull_docker_container ?
+        'https://depot.galaxyproject.org/singularity/mageck:0.5.9--py37h6bb024c_0':
+        'quay.io/biocontainers/mageck:0.5.9--py37h6bb024c_0' }"
+
+    input:
+    tuple val(meta), path(count_table)
+
+    output:
+    tuple val(meta), path("*.gene_summary.txt"), emit: gene_summary
```

Comment:
Could we align the emits for readability? Like e.g. https://github.com/nf-core/modules/blob/0dde9c00d859ed2cd09e91a97afded6031ba3589/modules/nf-core/deseq2/differential/main.nf#L15